tests/rt.sh

```diff
@@ -415,15 +415,15 @@ if [[ $TESTS_FILE =~ '35d' ]]; then
   TEST_35D=true
 fi
 
-BL_DATE=20210915
+BL_DATE=20210921
```

Comment:
Do answers change for this PR? I thought the input was updated but that the answers were the same?

Reply:
I am going to check, will revert it if not applicable.

Reply:
Do we need new baselines or not?

Reply:
The tests ran successfully with -c option. Now, I submitted the wave tests to be compared with the baseline, but my job is still in the queue.

Reply:
@DusanJovic-NOAA it seems I was wrong and the baseline does not need to be changed. Do you want me to revert it to 20210915?